includes/reader-revenue/woocommerce/class-woocommerce-connection.php

```diff
@@ -233,11 +234,10 @@ public static function get_last_successful_order( $customer ) {
 	 *
 	 * @param \WC_Order|int $order WooCommerce order or order ID.
 	 * @param bool|string   $payment_page_url Payment page URL. If not provided, checkout URL will be used.
-	 * @param bool          $is_new Whether the order is new and should count as the last payment amount.
 	 *
 	 * @return array Contact order metadata.
 	 */
-	public static function get_contact_order_metadata( $order, $payment_page_url = false, $is_new = false ) {
+	private static function get_contact_order_metadata( $order, $payment_page_url = false ) {
```

Comment:
I like that we're deprecating this parameter, but that seems to make "Last Payment Amount" and "Last Payment Date" values whichever order passed to `get_contact_from_customer()`. It's best to query it instead of relying on the passed order.

Reply:
Huh, good call. I was wondering if any of the contact metadata could rely on an order different than the last successful order (or the active subscription), but I don't think so. Applying this suggestion also fixes a potential bug – I'll update the PR description.